std_msgs/Header header
bool gripperState